src/OrbitGl/Batcher.h

```diff
@@ -117,6 +117,9 @@ class Batcher {
   Batcher(const Batcher&) = delete;
   Batcher(Batcher&&) = delete;
 
+  void PushTranslation(int x, int y, float z = 0.f);
```

Comment:
It's a bit strange to have different types per components, I see that callers have to static_cast<int> the x and y components, could it be easier to centralize that conversion from float to int in the batcher itself? The translation_stack_ and current_translation_ members both use floating points. Also, I think what we'd want is floorf() instead of static_cast<int> no?

Reply:
I agree. My intention was to clearly indicate that this would be pixel coordinates and any post-comma digits would be ignored anyways, but since we're using floats everywhere else, I've changed this now.